### PR TITLE
Redefined the variable of device,

### PR DESCRIPTION
**Pull request by jackcywang**
I've redefined the variable of device,I don't specify the gpu id.

### CODE REVIEW

File: train.py [hunk 1]
```diff
@@ -200,7 +200,7 @@ def freeze_backbone(m):
     epoch = 0
     best_loss = 1e5
     best_epoch = 0
-    step = max(0, last_step)
+    step = max(0, last_step)
```
Comment (jackcywang) [inline train.py:203]: sorry,I don't know I've changed this line of ocde

**Comment by zylo117** [comment]
About that. https://github.com/zylo117/Yet-Another-EfficientDet-Pytorch/pull/387#issuecomment-643283407

Maybe you haven't encountered that kinds of crashing that returns lots of meaningless and ambinguous cuda errors. You will need to re-run it on cpu to get some specific exception info.
And since `torch.cuda.is_available()` is always True on cuda servers, you can never debug it on cpu on cuda servers.

**Comment by jackcywang** [comment]
emm,maybe I don't consider the situation that who want to debug the model on cpu on the cuda servers,but in my opinion this kind of situation should be rare,maybe someone will do  it like that.And in factor,the `model.cuda()`equals to`model.to(torch.device('cuda'))`.maybe I haven't encountered that cuda crashing prblem in my previous projects

**Comment by jackcywang** [comment]
I withdraw the PR now, I will rerun it on my machine to  see what  is going!

**Pull request closed without merging**